runtime/jvmti/jvmtiThread.c

```diff
@@ -490,7 +475,8 @@ jvmtiGetThreadInfo(jvmtiEnv* env,
 			) {
 				/* java.lang.thread may not have a ref to vm thread. for example, after it gets terminated.
```

Comment:
thread -> Thread
Either change the period to a comma or capitalize the second sentence.